QEfficient/transformers/customop_utils.py

```diff
@@ -0,0 +1,32 @@
+custom_opset = onnxscript.values.Opset("com.qti.aisw.onnx", 1)
```

Comment:
Anything special or different to this customop? We can utilize the same `customop __init__.py` right?